book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_reader :rentals
```

Comment:
- [ ] Please add Getters and Setters (attr_accessor) for @rentals, not getters only (attr_reader) as you did so you can create a new rental.

Reply:
I have add the Getters and Setters using attr_accessor